### DISCUSSION TITLE
Obsidian template files

### DISCUSSION THREAD

**Discussion by atomicbird** [srid/emanote#227] [Q&A]
I have a couple of Obsidian templates that look similar to this:

```
---
aliases: 
tags: [  ]
title: {{title}}
---

# {{title}}
```

Emnote seems to not like the template variable in the front matter, printing an error of `User exception: BadInput "NonStringKey [Key \"title\"]"`.

Fair enough, I guess, but is there some way to have Emanote ignore my template files individually or have it skip the template directory? Ideally these files would be skipped completely, so Obsidian template vars would have no effect on Emanote results.

**Comment by srid** [comment]
I just opened https://github.com/srid/emanote/issues/228 for this.

**Reply by srid** [reply]
For now though perhaps you can put them in a folder starting with `.` (or `@`); they will be ignored by emanote.

**Reply by atomicbird** [reply]
Thanks! It looks like Obsidian also ignores names starting with `.` (so it doesn't see templates there), but `@` does the job.
